gazebo_ros/include/gazebo_ros/executor.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef GAZEBO_PLUGINS_EXECUTOR_HPP
+#define GAZEBO_PLUGINS_EXECUTOR_HPP
+
+#include <rclcpp/rclcpp.hpp>
+#include <gazebo/common/common.hh>
+
+namespace gazebo_ros
+{
+
+/// Executor run in a seperate thread to handle events from all #Node instances
```

Comment:
Could you use doxygen syntax on the whole documentation? i.e. `/// \brief`

Reply:
```\brief``` is implied with the first line of a doxygen block so my personal style preference is to omit it (it will still generate correctly)

Reply:
Interesting, I didn't know. I'm good leaving it out then.